src/icon/line/Report16.stories.tsx

```diff
@@ -1,13 +0,0 @@
-import * as React from 'react';
-
-import Report16 from './Report16';
-
-export const report16 = () => <Report16 />;
-
-export default {
-    title: 'Icon|Line|Report16',
-    component: Report16,
-    parameters: {
-        notes: "`import Report16 from 'box-ui-elements/es/icon/line/Report16';`",
```

Comment:
Does this no longer exist? I don't see it being removed in this PR.

Reply:
Looks like `Report16` got renamed to `AlertBubble16` which has it's corresponding stories file